packages/mui-joy/src/styles/types/typography.ts

```diff
@@ -0,0 +1,61 @@
+import * as CSS from 'csstype';
+import { CSSObject } from '@mui/system';
+
+type CSSProperties = CSS.Properties<number | string>;
+
+export interface FontSize {
+  default: CSSProperties['fontSize'];
+  xs: CSSProperties['fontSize'];
+  sm: CSSProperties['fontSize'];
+  md: CSSProperties['fontSize'];
+  lg: CSSProperties['fontSize'];
+  xl: CSSProperties['fontSize'];
+  xl2: CSSProperties['fontSize'];
+  xl3: CSSProperties['fontSize'];
+  xl4: CSSProperties['fontSize'];
+  xl5: CSSProperties['fontSize'];
+  xl6: CSSProperties['fontSize'];
+}
+
+export interface FontFamily {
+  default: CSSProperties['fontFamily'];
```

Comment:
```suggestion
  text: CSSProperties['fontFamily'];
```

Or whatever else is `default` :)

Reply:
I changed to `body`.